en/docs/setup/changing-the-hostname.md

```diff
@@ -10,7 +10,45 @@ Server.
     [server]
     hostname = "is.dev.wso2.com"
     ```
-
+    
+    1.  Configuring internal host name
+    
+        If you expect to configure a server hostname for internal API calls, it can be configured as follows. 
+        This is an optional config with the default value `localhost`. This configuration will be utilized to build the 
+        internal absolute url of a service endpoint, that will be consumed at places that will generate internal API calls.
+            
+        ``` toml
+        [server]
+        internal_hostname = "internal.wso2.is"
```

Comment:
Why does a user need to change the internal_hostname to a different one than "localhost"

Reply:
Updated the server hostname configuration steps with 48e547f